aws/policy/data-services.yaml

```diff
@@ -80,7 +80,14 @@ Statement:
       - rds:DeleteDBCluster
       - rds:DeleteDBParameterGroup
       - rds:DeleteDBSubnetGroup
-      - rds:DescribeDBParameters
+      - rds:DescribeDB*
```

Comment:
We want to move this into the first Sid, `AllowGlobalUnrestrictedResourceActionsWhichIncurNoFees`, otherwise we need to add more resources (tests got hung up on snapshots) and for Describe actions we mostly don't care.  The 3 Describe actions there now could be condensed down to a single wildcard.